src/parsers/manifestjson.js

```diff
@@ -203,4 +207,19 @@ export default class ManifestJSONParser {
       version: this.parsedJSON.version,
     };
   }
+
+  _checkForDuplicateKeys() {
+    try {
+      RJSON.parse(this._jsonString, { duplicate: true, warnings: true });
+    } catch (err) {
+      if (err.message.includes('Duplicate key')) {
```

Comment:
That’s why we have a test for it, if the underlying message changed we’d
know :-)